bin/ycsb

```diff
@@ -60,6 +60,7 @@ DATABASES = {
     "cassandra2-cql": "com.yahoo.ycsb.db.CassandraCQLClient",
     "couchbase"    : "com.yahoo.ycsb.db.CouchbaseClient",
     "couchbase2"   : "com.yahoo.ycsb.db.couchbase2.Couchbase2Client",
+    "documentdb"   : "com.yahoo.ycsb.db.DocumentDBClient",
```

Comment:
needs to be updated to new package location and should be `azuredocumentdb` instead of just `documentdb`